x-pack/legacy/plugins/maps/public/layers/sources/es_pew_pew_source/es_pew_pew_source.js

```diff
@@ -126,22 +125,22 @@ export class ESPewPewSource extends AbstractESAggSource {
   createDefaultLayer(options) {
     const styleDescriptor = VectorStyle.createDescriptor({
       [vectorStyles.LINE_COLOR]: {
-        type: VectorStyle.STYLE_TYPE.DYNAMIC,
+        type: DynamicStyleProperty.type,
         options: {
           field: {
-            label: COUNT_PROP_LABEL,
-            name: COUNT_PROP_NAME,
+            label: AbstractESAggSource.COUNT_PROP_LABEL,
+            name: AbstractESAggSource.COUNT_PROP_NANE,
             origin: SOURCE_DATA_ID_ORIGIN
           },
           color: 'Blues'
         }
       },
       [vectorStyles.LINE_WIDTH]: {
-        type: VectorStyle.STYLE_TYPE.DYNAMIC,
+        type: DynamicStyleProperty.type,
         options: {
           field: {
-            label: COUNT_PROP_LABEL,
-            name: COUNT_PROP_NAME,
+            label: AbstractESAggSource.COUNT_PROP_LABEL,
```

Comment:
AbstractESAggSource does not have properties COUNT_PROP_LABEL or COUNT_PROP_NANE (and COUNT_PROP_NANE is a typo, NANE). Should just import directly from constants.

Breaks pew-pew source
<img width="200" alt="Screen Shot 2019-11-19 at 9 05 41 AM" src="https://user-images.githubusercontent.com/373691/69163640-cd673580-0aab-11ea-8394-19936d44e087.png">